deselected_tests.yaml

```diff
@@ -1105,6 +1105,8 @@ gpu:
   - tests/test_common.py::test_estimators[RFECV(estimator=LogisticRegression(C=1))-
   - tests/test_common.py::test_estimators[SelfTrainingClassifier(base_estimator=LogisticRegression(C=1))-
   - tests/test_common.py::test_estimators[SequentialFeatureSelector(estimator=LogisticRegression(C=1))-
+  # 2024.0 update
```

Comment:
Comment is not informative (update of what, what is the actual deselection reason, etc.)